format.md

```diff
@@ -0,0 +1,138 @@
+# About this document
+
+This document attempts to document the format currently implemented by binjs-fbssdc, also known as "context 0.1".
+
+# Global structure
+
+```
+Stream ::= MagicHeader BrotliBody
+```
+
+# Magic Header
+
+The magic header serves to identify the format.
+
+```
+MagicHeader ::= "\x89BJS\r\n\0\n" FormatVersion
+FormatVersion ::= 0b00000010
```

Comment:
would be nice to say it means `2` (not varnum with `1`)

Reply:
my comment was wrong that `varint` used in this Python script (uses highest bit for flag) is different than `varnum` in entropy (that uses lowest bit for flag).
so either way this means `2`.